mungegithub/mungers/publisher.go

```diff
@@ -113,45 +150,49 @@ func clone(dst string, dstURL string) error {
 
 // construct checks out the source repo, copy the contents to the destination,
 // returns a commit message snippet and error.
-func construct(base, org string, src, dst coordinate) (string, error) {
+// this function is only used when publishing client-go.
+func (p *PublisherMunger) construct(src, dst coordinate) (string, error) {
+	base := p.baseDir
+	org := p.githubConfig.Org
 	srcRepoRoot := filepath.Join(base, src.repo)
 	srcDir := filepath.Join(base, src.repo, src.dir)
-	dstRepoRoot := filepath.Join(base, dst.repo)
+	dstRepoRoot := filepath.Join(base, goWorkspaceStructure, dst.repo)
 	curDir, err := os.Getwd()
 	if err != nil {
-		glog.Infof("Getwd failed")
+		p.plog.Infof("Getwd failed")
```

Comment:
You should actually print out the error in all of these, so people will know what to do.